.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 .env
+indexer-events/src/.DS_Store
```

Comment:
We try to avoid adding personal stuff to gitignore. Instead of that, it's better to have gitignore on your local machine:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
FYI you can use the `.git/info/exclude` file to add ignore patterns which are specific to your machine